blog/2015/11/swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,436 @@
+---
+layout: blog
+root: ../../..
+author: Daniel Chen
+title: "SWDCRetreat2015 - Round-table discussion: Software Carpentry as a University Course Recap"
+date: 2015-11-16
+time: "11:30:00"
+category: ["Community", "Teaching", "SWDCRetreat"]
+---
+<!-- start excerpt -->
+The <a href='http://swcarpentry.github.io/instructor-retreat-2015/'> inaugural Software Carpentry + Data Carpentry Instructor & Helper Retreat</a> is over!
+It was a long day, packed with tutorials, demos, and discussions.
+<a href='https://twitter.com/chendaniely'>I (Daniel Chen)</a>
+led a round table discussion with
+<a href='https://twitter.com/TiffanyTimbers'>Tiffany Timbers</a> and
+<a href='https://twitter.com/JennyBryan'>Jenny Bryan</a>.
+You can <a href='https://plus.google.com/events/c41eaub933br0odpb385rcg9j00'>watch</a> the discussion and/or
+<a href='http://pad.software-carpentry.org/swc-instructor-retreat-2015-7PM-UTC'>read</a> the notes on etherpad.
+
+<!-- end excerpt -->
+
+First, thanks to everyone who joined the discussion:
+<ul>
+  <li>Tiffany Timbers / @TiffanyTimbers</li>
+  <li>Jenny Bryan / @JennyBryan </li>
+  <li>Daniel Chen / @chendaniely </li>
+  <li>Camille Avestruz</li>
+  <li>David LeBauer</li>
+  <li>Cam Macdonell</li>
+  <li>Drew Tyre / @atiretoo</li>
+  <li>Rayna Harris / @raynamharris and the +5 from <a href='https://twitter.com/choleness/status/665613152181092352'>Austin Texas</a> including: Rebecca Tarvin / @frogsicles</li>
+  <li>Jon Pipitone</li>
+  <li>John Moreau</li>
+  <li>Gayathri Swaminathan</li>
+  <li>Nichole Bennett</li>
+  <li>Tracy Teal</li>
+  <li>Raniere Silva</li>
+  <li>Sarah Stevens</li>
+  <li>Lex Nederbragt</li>
+  <li>Kate Hertweck</li>
+  <li>Greg Wilson</li>
+  <li>Becca Tarvin</li>
+  <li>Azalee Bostroem</li>
+</ul>
+
+Hopefully I got everyone!
+
+<h2>Motivation</h2>
+Last year I <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002202.html'>asked</a> the <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/'>discuss list</a>
+if anyone had any experience with turing the <a href='http://software-carpentry.org/lessons.html'>software-carpentry lessons</a> into a university course.
+Bill Mills <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002203.html'>mentioned</a> that two main hurdles revolve around politics and curriculum control.
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002204.html'>Ethan White</a>
+and <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002211.html'>Michael Jackson</a>went on to describe their own experiences with settin up a course.
```

Comment:
“`Michael Jackson</a>went`” → “`Michael Jackson</a> went`” (insert space after the `</a>`).